docs/Training-ML-Agents.md

```diff
@@ -181,8 +181,8 @@ and explains the possible settings for each.
 
 **NOTE:** The configuration file format has been changed from 0.17.0 and onwards. To convert
 an old set of configuration files (trainer config, curriculum, and sampler files) to the new
-format, a script has been provided. Run `python config/upgrade_config.py -h` in your  console
-to see the script's usage.
+format, a script has been provided. Run `python -m mlagents.trainers.upgrade_config -h` in your
+console to see the script's usage.
```

Comment:
I tried to convert an already converted file using the script and got this error : 
```
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/vincentpierre/Documents/ml-agents/ml-agents/mlagents/trainers/upgrade_config.py", line 119, in <module>
    main()
  File "/Users/vincentpierre/Documents/ml-agents/ml-agents/mlagents/trainers/upgrade_config.py", line 99, in main
    behavior_config_dict = convert_behaviors(old_config)
  File "/Users/vincentpierre/Documents/ml-agents/ml-agents/mlagents/trainers/upgrade_config.py", line 21, in convert_behaviors
    trainer_type = config["trainer"]
KeyError: 'trainer'
```
I think we should try catch some key errors and raise more graceful errors.

Reply:
Added exceptions for the two potential KeyErrors (`trainer` and `use_recurrent`).